fortran/example/README.md

```diff
@@ -0,0 +1,75 @@
+# How to use spglib example file in Fortran
+
+This is the README file for spglib examples for Fortran.
+
+## Examples of using spglib in a cmake project
+
+See the local [`CMakeLists.txt`](CMakeLists.txt) file for how to use
+spglib in a cmake project, using either native cmake options or pkg-config.
+
+## Examples of linking to spglib manually
+
+To manually compile `example.f90`:
+
+```console
+gfortran example.f90 ../../lib/libspglib_f08.a ../../lib/libsymspg.a -I../../include
+```
+
+Here it is assumed that spglib library was built by
+```console
+cmake -B build -DCMAKE_INSTALL_PREFIX=. -DSPGLIB_WITH_Fortran=on -DSPGLIB_SHARED_LIBS=off -DSPGLIB_WITH_TESTS=off .
+cmake --build build
+cmake --install build
+```
+
+## Concept
```

Comment:
Similarly, Concept should be in top-level, here we specify more language specific notes.

Reply:
This is treated at 5800eb0.